extensions/amp-base-carousel/0.1/test-e2e/test-arrows.js

```diff
@@ -23,102 +23,85 @@ import {
   getSlide,
 } from './helpers';
 
-
-describes.endtoend('AMP carousel arrows', {
+/** The total number of slides in the carousel */
+const SLIDE_COUNT = 7;
+const pageWidth = 600;
+const pageHeight = 600;
+
+describes.endtoend('AMP carousel arrows when non-looping', {
```

Comment:
rather than having two top level describes, I'd rather have separate test files. It does feel unfortunate to give up the ability to nest test cases here though.

Reply:
I can wrap it in a `describe('AMP carousel arrows', () => {})`. I was mainly trying to avoid extra indentation, but the duplication is not great

Reply:
If it's OK, I'll address this up in a follow-up PR to keep review simple. `test-mixed-lengths.js` also needs to be addressed for this

Reply:
#21243